_posts/2022-01-21-hhvm-4.145.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: "HHVM 4.144"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.146 is released! This release marks the end of support of 4.137;
+4.138, 4.139, 4.140, 4.143, 4.144 remain supported, as do the 4.102 and 4.128 LTS releases.
+4.141 and 4.142 were cancelled due to the holiday season.
+
+
+# Highlights
+
+- Support throw on dynamic, e.g. `function foo(dynamic $d): void { throw $d; }`
+- Add support for attributes on class constants
+- Strictify several built-in functions
+- Several bug fixes
```

Comment:
Remove or expand; this isn't useful by itself

Reply:
Removed